src/main/java/no/nav/familie/integrasjoner/client/rest/OppgaveRestClient.kt

```diff
@@ -88,6 +88,66 @@ class OppgaveRestClient(@Value("\${OPPGAVE_URL}") private val oppgaveBaseUrl: UR
         return finnAlleOppgaver()
     }
 
+    fun finnOppgaverV2(tema: String,
+                       behandlingstema: String?,
+                       oppgavetype: String?,
+                       tildeltEnhetsnr: String?,
+                       tilordnetRessurs: String?,
+                       journalpostId: String?,
+                       opprettetFom: String?,
+                       opprettetTom: String?,
+                       fristFom: String?,
+                       fristTom: String?,
+                       aktivDatoFom: String?,
+                       aktivDatoTom: String?,
+                       limit: Long?,
+                       offset: Long?): FinnOppgaveResponseDto {
+
+        val limitMotOppgave = 50
```

Comment:
Tenker det er fint om man kan spesifisere denne limiten i requesten istedet for at den er hardkodet :)

Reply:
Nei, dette er maxgrensen mot Oppgave og skal ikke eksponeres ut. Vi har vår egen limit, spesifisert i controlleren.

Reply:
Ok, supert!